### ISSUE TITLE
cannot install lottie-ios@3.4.0

### ISSUE THREAD

**Issue by ethanneff** [lottie-react-native/lottie-react-native#906]
### Description

Fails on 3.4.0
<img width="299" alt="Screen Shot 2022-07-22 at 11 28 50 AM" src="https://user-images.githubusercontent.com/2933593/180501740-a5b8aece-86d6-4735-b98b-e498d9d7a1c1.png">

Works when downgrading to 3.2.3

<img width="301" alt="Screen Shot 2022-07-22 at 11 28 55 AM" src="https://user-images.githubusercontent.com/2933593/180501753-3e6b06fd-a064-4a8a-a3b9-bfb47eecf102.png">

We need to revert #901 

### Steps to Reproduce

Follow Readme for the first time. 
https://github.com/lottie-react-native/lottie-react-native#ios-and-android

### Versions

8.8.0

**Comment by matinzd** [comment]
Same issue here:

```
[!] CocoaPods could not find compatible versions for pod "lottie-ios":
  In snapshot (Podfile.lock):
    lottie-ios (from `../node_modules/lottie-ios`)

  In Podfile:
    lottie-ios (from `../node_modules/lottie-ios`)

    lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.3, which depends on
      lottie-ios (~> 3.2.3)
```

**Comment by Insaev** [comment]
Same issue too

**Comment by matinzd** [comment]
It seems that a new version is pending due to this pr: https://github.com/lottie-react-native/lottie-react-native/pull/878

**Comment by cq0702** [comment]
Same issue.
3.2.3 doesn't work for me.
yarn add lottie-react-native lottie-ios@3.2.3

**Comment by Deepak-HDFC** [comment]
node modules podspec has dependency on 3.2.3 version of lottie-ios on lottie-react-native version 5.1.3
<img width="854" alt="Screenshot 2022-07-25 at 5 58 59 PM" src="https://user-images.githubusercontent.com/98941610/180777492-0ca48c25-db19-4f9a-84e1-1cae5883e7b4.png">

**Comment by shensven** [comment]
same problem

**Comment by lukaspechar** [comment]
> node modules podspec has dependency on 3.2.3 version of lottie-ios on lottie-react-native version 5.1.3 <img alt="Screenshot 2022-07-25 at 5 58 59 PM" width="854" src="https://user-images.githubusercontent.com/98941610/180777492-0ca48c25-db19-4f9a-84e1-1cae5883e7b4.png"> Updating the dependency on podspec to 3.4.0 it works.

This is the only working solution at the moment. The latest version still has the bug. Thanks for this. You should ask the team to have it merged

**Comment by archcorsair** [comment]
Also experiencing this using the recommended combination: `lottie-react-native@latest` `lottie-ios@3.4.0`
```
[!] CocoaPods could not find compatible versions for pod "lottie-ios":
  In snapshot (Podfile.lock):
    lottie-ios

  In Podfile:
    lottie-ios (from `../node_modules/lottie-ios`)

    lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.3, which depends on
      lottie-ios (~> 3.2.3)
```

**Comment by ceyanesb** [comment]
> Also experiencing this using the recommended combination: `lottie-react-native@latest` `lottie-ios@3.4.0` Updating `lottie-react-native.podspec` did solve the issue for now!
> 
> ```
> [!] CocoaPods could not find compatible versions for pod "lottie-ios":
>   In snapshot (Podfile.lock):
>     lottie-ios
> 
>   In Podfile:
>     lottie-ios (from `../node_modules/lottie-ios`)
> 
>     lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.3, which depends on
>       lottie-ios (~> 3.2.3)
> ```

Yes, just updating this will work, please release a new update!

**Comment by dialex** [comment]
They mentioned this issue a while go, maybe we are closer to getting it merged ☝️

I inspected that PR, [it doesn't actually change the line that has the bug](https://github.com/lottie-react-native/lottie-react-native/pull/878/files#diff-d7b0b62548e1dfb2ae4c8d5a04febd3c456a3dbe3d0f8b1384bd973457612fd9L25), so maybe that line is already fixed in `master`, but they have not released it! Which means, merging that PR won't be enough to fix our issue. Luckily for us, [they noticed it](https://github.com/lottie-react-native/lottie-react-native/pull/878#issuecomment-1204774538) (phew), so we need to wait for the PR merge **and** release. _Hold still my beating_ ❤️ 😬

**Comment by LonelyCpp** [comment]
They've updated the readme on master because the the update commit has landed on master (but has not been released yet)

if you see the [readme on the v5.1.3 tag](https://github.com/lottie-react-native/lottie-react-native/tree/v5.1.3), it tells us to install `lottie-ios@3.2.3`

so if you're installing v5.1.3 - run 
```bash
yarn add lottie-react-native
yarn add lottie-ios@3.2.3
```

**Comment by matinzd** [comment]
Please try installing 5.1.4. 
You can see the changelog here:
https://github.com/lottie-react-native/lottie-react-native/releases/tag/v5.1.4

**Comment by shamilovtim** [comment]
This is unresolved, even with 5.1.4:

```
Command `pod install` failed.
└─ Cause: CocoaPods could not find compatible versions for pod "lottie-ios":
  In Podfile:
    lottie-ios (from `../node_modules/lottie-ios`)

    lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.4, which depends on
      lottie-ios (~> 3.4.0)
```

**Comment by matinzd** [comment]
> ~This is unresolved, even with 5.1.4~
> 
> users must use exactly lottie-ios@3.4.0 or earlier

Yes

**Comment by Murphyst** [comment]
make sure that both lottie-ios and lottie-react-native have identical dependency versions.

**Comment by kockok** [comment]
I'm using    
```
 "lottie-react-native": "^5.1.5",
    "lottie-ios": "^4.1.3",
```
And the problem still there:

```
Command `pod install` failed.
└─ Cause: CocoaPods could not find compatible versions for pod "lottie-ios":
  In Podfile:
    lottie-ios (from `../node_modules/lottie-ios`)

    lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.5, which depends on
      lottie-ios (~> 3.4.0)
```

I can see that the readme page only states that `<= 5.1.4` will need to install `lottie-ios 3.4.0`.

Just wondering if version 5.1.5 still requires users to install `lottie-ios 3.4.0`.

**Comment by matinzd** [comment]
> I'm using    
> 
> ```
> 
>  "lottie-react-native": "^5.1.5",
> 
>     "lottie-ios": "^4.1.3",
> 
> ```
> 
> And the problem still there:
> 
> 
> 
> ```
> 
> Command `pod install` failed.
> 
> └─ Cause: CocoaPods could not find compatible versions for pod "lottie-ios":
> 
>   In Podfile:
> 
>     lottie-ios (from `../node_modules/lottie-ios`)
> 
> 
> 
>     lottie-react-native (from `../node_modules/lottie-react-native`) was resolved to 5.1.5, which depends on
> 
>       lottie-ios (~> 3.4.0)
> 
> ```
> 
> 
> 
> I can see that the readme page only states that `<= 5.1.4` will need to install `lottie-ios 3.4.0`.
> 
> 
> 
> Just wondering if version 5.1.5 still requires users to install `lottie-ios 3.4.0`.
> 
> 

You do not need to install lottie-ios separately after 5.1.4. 

Run `pod update` to update your cocoapods repositories or clear your cache or try it with a new project. 

I will close issue as I could not reproduce on a new fresh project.

**Comment by archcorsair** [comment]
> You do not need to install lottie-ios separately after 5.1.4.
> 
> Run `pod update` to update your cocoapods repositories or clear your cache or try it with a new project.

This is the final and correct solution, thank you!

**Comment by dennis-gonzales** [comment]
> > You do not need to install lottie-ios separately after 5.1.4.
> > Run `pod update` to update your cocoapods repositories or clear your cache or try it with a new project.
> 
> This is the final and correct solution, thank you!

thanks

**Comment by Durotemy** [comment]
I tried to  do a local build 
eas build --platform ios --local
i get this error
Fastlane is not available, make sure it's installed and in your PATH
spawn fastlane ENOENT
npx exited with non-zero code: 1


But when I try to do  a build for ios i get this error
eas build -p android --profile production

[!] CocoaPods could not find compatible versions for pod "lottie-ios":
  In Podfile:
    lottie-ios (from ../node_modules/lottie-ios)
    lottie-react-native (from ../node_modules/lottie-react-native) was resolved to 6.5.1, which depends on
      lottie-ios (~> 4.3.3)
[!] <PBXResourcesBuildPhase UUID=13B07F8E1A680F5B00A75B9A> attempted to initialize an object with an unknown UUID. DFC5960ABF794B898B00C404 for attribute: files. This can be the result of a merge and the unknown UUID is being discarded.
Error: Compatible versions of some pods could not be resolved.
You are seeing this error because either:

**Comment by AkibShaih** [comment]
> pod update

Worked for me
